### PR TITLE
[Manifest] fix version for MLMD store

### PR DESCRIPTION
**Pull request by numerology**
It seems like they'll remove the 'v' in the version tag.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/3237)

### CODE REVIEW

**Comment by numerology** [comment]
/assign @dushyanthsc 
/assign @rmgogogo 

FYI

File: .release.cloudbuild.yaml [hunk 1]
```diff
@@ -243,14 +243,14 @@ steps:
     docker push gcr.io/ml-pipeline/google/pipelines-test/visualizationserver:$(cat /workspace/mm.ver)
 
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['pull', 'gcr.io/tfx-oss-public/ml_metadata_store_server:v0.21.1']
```
Comment (randmind) [inline .release.cloudbuild.yaml]: https://github.com/kubeflow/pipelines/blob/master/.release.cloudbuild.yaml#L246

Comment (rmgogogo) [inline .release.cloudbuild.yaml]: https://github.com/kubeflow/pipelines/blob/master/.release.cloudbuild.yaml#L246

Comment (numerology) [reply .release.cloudbuild.yaml]: Can you explain a bit more? A little bit of confused by the link

Comment (rmgogogo) [reply .release.cloudbuild.yaml]: I means this part also need changes

Comment (numerology) [reply .release.cloudbuild.yaml]: I guess you meant https://github.com/kubeflow/pipelines/blob/master/.release.cloudbuild.yaml#L263 ?

Comment (numerology) [reply .release.cloudbuild.yaml]: Changed accordingly

**Pull request merged**